specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Unit
+
+The Pricing Unit is a measurement unit specified by the provider, serving as the foundation (a.k.a. basis) for calculating unit prices associated with using a specific resource and/or service. Common examples include the number of hours a compute appliance was running for a given period, the number of gigabytes * the number of hours (gigabyte-hours) for a storage appliance, or a cumulative count of requests for a network appliance or API service. For certain services providers can choose block pricing i.e., million requests as a base unit.
+PricingUnit is complementary to and describes how [QuantityInPricingUnit](#quantityinpricingunit) is measured.
+
```

Comment:
```suggestion
The Pricing Unit for a particular line item may differ from the published Usage Unit (and complementary Quantity in Usage Unit) when providers use different units, unit increments, or columns for calculating cost. 
```

Reply:
I've added a remark regarding the UsageUnit/PricingUnit within Usage Unit PR as well. This seems a bit unclear and implies that there are UCs in which providers use different columns for cost calculation?!

Suggestions:
- focus on differing purposes (primary distinction?!) 
- avoid mentioning that the values may differ and when that may be the case as covering those UCs seams a bit challenging

We might consider something like this: 
_While PricingUnit and QuantityInPricing may appear to relate to resource and/or service consumption, their primary focus is on pricing and cost, so it's essential not to confuse them with UsageUnit and QuantityInUsageUnit, which are dedicated to tracking actual resource and service consumption._

Rephrase? Simplify? Definitely!

Reply:
Discussed at a meeting and agreed to emphasize the difference in purpose rather than mentioning that UsageUnit and PricingUnit may differ and in which cases. A similar note or remark will be included in the definition of all four columns - PricingUnit, QuantityInPricingUnit, UsageUnit, QuantityInPricingUnit.